src/handlers/payout/action.ts

```diff
@@ -220,9 +221,17 @@ export const incentivesCalculation = async (): Promise<IncentivesCalculationResu
     throw new Error(errMsg);
   }
 
+  const permitComments = comments.filter((content) => content.body.includes("https://pay.ubq.fi?claim=") && content.user.type == UserType.Bot);
```

Comment:
We need to stop relying on `"https://pay.ubq.fi?claim="` but I guess it might make more sense to handle as part of another issue.

Reply:
> but I guess it might make more sense to handle as part of another issue

Parsing the URL should work right?

Reply:
We have another issue where it uses `new URL()` to make sense of it 

The final implementation will rely on comment metadata which is an html comment inside of the bot comment